src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -42,6 +42,7 @@ public function process(ContainerBuilder $container)
         foreach ($commandServices as $id => $tags) {
             $definition = $container->getDefinition($id);
             $definition->addTag('container.no_preload');
+            /** @var class-string<Command> $class */
```

Comment:
This is a "lie", which is [not allowed per our phpdoc standards](https://symfony.com/doc/current/contributing/code/standards.html#documentation).

Nothing in the contract guarantees that `$class` is a class string of `Command`. Adding this PHPdoc means we'll miss the edge-cases where this is not the case.

The same thing applies to the other inline PHPdocs of this PR.

Reply:
I've removed the inline PHPdocs.